qa/L0_infer/test.sh

```diff
@@ -57,18 +58,21 @@ for TARGET in cpu gpu; do
         fi
         # set strict readiness=false on CPU-only device to allow
         # unsuccessful load of TensorRT plans, which require GPU.
-        SERVER_ARGS="--model-store=$DATADIR --strict-readiness=false"
+        SERVER_ARGS="--model-store=$DATADIR --exit-timeout-secs=120 --strict-readiness=false"
     fi
 
     SERVER_LOG=$SERVER_LOG_BASE.${TARGET}.log
     CLIENT_LOG=$CLIENT_LOG_BASE.${TARGET}.log
 
     rm -fr models && \
         cp -r /data/inferenceserver/qa_model_repository models && \
+        cp -r /data/inferenceserver/qa_ensemble_model_repository/qa_model_repository/* models/. && \
         cp -r ../custom_models/custom_float32_* models/. && \
         cp -r ../custom_models/custom_int32_* models/. && \
         cp -r ../custom_models/custom_nobatch_* models/.
 
+    create_nop_modelfile ../L0_infer_reshape/libidentity.so `pwd`/models
```

Comment:
Don't copy from L0_infer_reshape. That is very brittle. If you need libidentity.so in the test then change Dockerfile.QA to copy it in when create the QA container.